Reneiski_Homework1/ReneiskiSelenium11/PageObjects/Pages/RegFormPage.cs

```diff
@@ -0,0 +1,32 @@
+﻿using OpenQA.Selenium;
+using ReneiskiSelenium11.Common.WebElements;
+using ReneiskiSelenium11.Data;
+
+namespace ReneiskiSelenium11.PageObjects.Pages
+{
+    public class RegFormPage : WebTablePage
```

Comment:
it's better to write full name of the page in the class name

Reply:
Fixed